propel.yaml

```diff
@@ -0,0 +1,33 @@
+services:
+- name: reaction-core
+  certificate_arn: arn:aws:acm:us-west-2:773713188930:certificate/c2979a7a-7b84-43ed-b3a7-24b0256a1b9f
+  root_domain: staging.reactioncommerce.com
+  dns_name: reaction-core
+  desired_task_count: 2
+  min_task_count: 0
+  max_task_count: 2
+  min_healthy_percent: 100
+  max_percent: 200
+  alb_listener_port: 80
+  alb_listener_path: /
+  alb_health_check_path: /health
+  task-definition:
+    name: staging-core
+    containers:
+    - name: core
+      cpu: 500
+      memory: 1800
+      port_mappings:
+      - container_port: 3000
+        host_port: 3000
+      image: reactioncommerce/reaction
+      image_tag: release-2.0.0-rc.5
```

Comment:
So this will need to be updated every time we make a new release branch?

Reply:
The image_tag is used only if you deploy using propel from the command line. CircleCI deployments will automatically use the SHA1 of the newly built Docker image.